simtools/db/db_handler.py

```diff
@@ -936,26 +928,15 @@ def add_tagged_version(
         ----------
         released_version: str
             The version name to set as "Released"
-        released_label: str
-            The released version name as label.
-        latest_version: str
-            The version name to set as "Latest"
-        latest_label: str
-            The latest version name as label.
+        tags: dict
+            The version tags consisting of tag name and version name.
```

Comment:
I would like to avoid this - as changes of the format would mean also to remember to change the docstrings.

simtools is very tighly interwoven with the model parameter databases and repositories. There is no need to put documentation in both places.